frost-core/src/frost/keys.rs

```diff
@@ -275,7 +279,11 @@ where
             return Err("SecretShare is invalid.");
         }
 
-        Ok(())
+        let group_public = VerifyingKey {
+            element: self.commitment.0[0].0,
```

Comment:
Hm this API isn't the best (getting the group verifying key out of `VerifiableSecretSharingCommitment`) but whatever

Reply:
Do you mean conceptually, or this specific code? I agree `self.commitment.0[0].0` is terribly ugly and it's coming up a lot in the DKG code too. Maybe I can add a `impl From<&VerifiableSecretSharingCommitment> for VerifyingKey`?

Reply:
@dconnolly what do you think?

Reply:
Ah yes this is just a style nit, we can try to make better in the DKG work perhaps, not a blocker